scio-examples/src/main/scala/com/spotify/scio/examples/extra/JavaConvertersExample.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2018 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio.examples.extra
+
+import com.spotify.scio.ContextAndArgs
+import com.spotify.scio.JavaConverters._
+import org.apache.beam.sdk.io.TextIO
+import org.apache.beam.sdk.options.ValueProvider.StaticValueProvider
+
+object JavaConvertersExample {
+  def main(cmdlineArgs: Array[String]): Unit = {
+    val (sc, args) = ContextAndArgs(cmdlineArgs)
+
+    val output = args("output")
+    val t: TextIO.TypedWrite[String, Void] = TextIO.writeCustomType()
+    val transform = args("converter") match {
```

Comment:
To avoid a bunch of separate example pipelines, thought it would be best to parameterize the Java Conversion applied to the TextIO.typedWrite transform. LMK if you prefer another approach